i18n/validate.py

```diff
@@ -26,8 +26,12 @@ def validate_po_files(root, report_empty=False):
     for dirpath, __, filenames in os.walk(root):
         for name in filenames:
             __, ext = os.path.splitext(name)
-            if ext.lower() == '.po':
-                filename = os.path.join(dirpath, name)
+            filename = os.path.join(dirpath, name)
+
+            # validate only .po files that are not product of a merge (see generate.py)
```

Comment:
Sorry to be picky, but would you mind using English sentence style for comments?  Capitalize the first letter, and use periods at the end of sentences.  It makes the comments easier to read and understand.